source/appModules/winword.py

```diff
@@ -54,7 +54,9 @@ class WinwordWordDocument(WordDocument):
 	def _get_description(self) -> str:
 		try:
 			curView = self.WinwordWindowObject.view.Type
```

Comment:
Could you please save the super description in a variable in the method? Also ensure that you don't call _get_description() but just get the description property
```suggestion
			curView = self.WinwordWindowObject.view.Type
			description = super().description
```